engine/src/witness/sol.rs

```diff
@@ -132,6 +127,32 @@ impl VoterApi<SolanaNonceTracking> for SolanaNonceTrackingVoter {
 	}
 }
 
+#[derive(Clone)]
+struct SolanaEgressWitnessingVoter {
+	client: SolRetryRpcClient,
+}
+
+#[async_trait::async_trait]
+impl VoterApi<SolanaEgressWitnessing> for SolanaEgressWitnessingVoter {
+	async fn vote(
+		&self,
+		_settings: <SolanaEgressWitnessing as ElectoralSystem>::ElectoralSettings,
+		signature: <SolanaEgressWitnessing as ElectoralSystem>::ElectionProperties,
+	) -> Result<
+		<<SolanaEgressWitnessing as ElectoralSystem>::Vote as VoteStorage>::Vote,
+		anyhow::Error,
+	> {
+		tracing::info!("Requesting success witnesses for {:?}", signature);
```

Comment:
Does this need to be `info`? Maybe debug is enough? (I have no idea how frequently this might be polled.

Reply:
I was thinking of removing it entirely - the RPC is logging as we go

Reply:
removed it